qadence/register.py

```diff
@@ -77,27 +89,43 @@ def from_coordinates(
         coords: list[tuple],
         lattice: LatticeTopology | str = LatticeTopology.ARBITRARY,
         spacing: float | None = None,
+        device_specs: RydbergDevice = IdealDevice(),
```

Comment:
Would it make sense to create a module wide constant for this ? Instead of instantiating everywhere in method signatures ?

Reply:
Yes indeed